packages/mdc-checkbox/_mixins.scss

```diff
@@ -147,8 +146,6 @@ $mdc-checkbox-ripple-target: ".mdc-checkbox__ripple";
       @include mdc-checkbox__native-control_;
     }
 
-    @include mdc-checkbox-touch-target($mdc-checkbox-ripple-size, $query);
```

Comment:
These styles weren't getting applied because of invalid selector. i.e., `.mdc-checkbox__native-control .mdc-checkbox__native-control { ...}`

Reply:
Nice, thanks for fixing that.